copy/copy.go

```diff
@@ -420,7 +422,7 @@ func (c *copier) copyMultipleImages(ctx context.Context, policyContext *signatur
 	if err != nil {
 		return nil, fmt.Errorf("reading manifest list: %w", err)
 	}
-	originalList, err := manifest.ListFromBlob(manifestList, manifestType)
+	originalList, err := internalManifest.ListInternalFromBlob(manifestList, manifestType)
 	if err != nil {
 		return nil, fmt.Errorf("parsing manifest list %q: %w", string(manifestList), err)
 	}
```

Comment:
Just below, `updatedList.DoNothing()` will need to work for us to be able to add/remove Zstd variants.